lib/lanuch.ex

```diff
@@ -0,0 +1,87 @@
+defmodule Membrane.OpenTelemetry.Plugs.Launch do
+  @moduledoc """
+  Attaches OpenTelemetry spans describing events during components launch, since `handle_init` until `handle_end_of_stream`
+  """
+
+  alias __MODULE__.{HandlerFunctions, ETSWrapper}
+
+  @all_callbacks [
+                   Membrane.Pipeline,
+                   Membrane.Bin,
+                   Membrane.Element.Base,
+                   Membrane.Element.WithInputPads,
+                   Membrane.Element.WithOutputPads
+                 ]
+                 |> Enum.flat_map(fn module -> module.behaviour_info(:callbacks) end)
+                 |> Keyword.keys()
+                 |> Enum.uniq()
+                 |> List.delete(:__struct__)
```

Comment:
Perhaps it would be good to make list of all telemetry events defined in membrane_core available to obtain with some function